core/kits/documents/tabs/colors-and-typography.php

```diff
@@ -35,14 +36,17 @@ public function register_tab_controls() {
 			'title',
 			[
 				'type' => Controls_Manager::TEXT,
+				'label_block' => true,
 			]
 		);
 
 		$repeater->add_control(
 			'color',
 			[
 				'type' => Controls_Manager::COLOR,
+				'label_block' => true,
 				'dynamic' => [],
+				'global_settings' => true,
```

Comment:
What is that?

Reply:
This is temporary. It is for the color control handler, to not add the Global Control Select behavior in the Global Settings panel page.
We will need to handle this when we move the Global Control Select behavior to attach itself to the global controls with the filter in the `behaviors()` method.